x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -72,6 +72,7 @@ export const signalRulesAlertType = ({
         riskScore: schema.number(),
         severity: schema.string(),
         threat: schema.nullable(schema.arrayOf(schema.object({}, { allowUnknowns: true }))),
+        throttle: schema.string(),
```

Comment:
I don't think you need this here. This is duplicating throttle as it is a first class parameter within the alerting framework. We don't want to duplicate anything from the alerting framework within the params and have duplicate values being stored in the data store and issues with one being updated versus the other not being the same type of value.

I will add another comment in another section of the code about this where you update the two differently.

You will want to remove this from params and use just the value from the alerting framework.